src/xcode/ENA/ENAUITests/AppInformation/ENAUITestsAppInformation.swift

```diff
@@ -85,6 +85,7 @@ class ENAUITests_02_AppInformation: XCTestCase {
 		XCTAssert(app.cells["AppStrings.AppInformation.faqNavigation"].waitForExistence(timeout: 5.0))
 		app.cells["AppStrings.AppInformation.faqNavigation"].tap()
 
+		// the following test will fail if device language is not English
```

Comment:
To be solved(?) in [#3144](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3144)

Reply:
Nope. As seen in the results it's creating screenshots for all languages. 
Handling taps on system buttons such as 'Done' is not in scope of this ticket. That's something we cover in 3144.